libs/libs/src/cookies/getCookie.ts

```diff
@@ -22,9 +22,11 @@ export const getCookie = ({
 
 	if (cookieVal.length > 0) {
 		if (shouldMemoize) {
-			memoizedCookies.set(name, cookieVal[0]);
+			// TODO: extract this to a function
+			memoizedCookies.set(name, cookieVal[0] ? cookieVal[0] : '');
```

Comment:
Argument of type 'string | undefined' is not assignable to parameter of type 'string'.
  Type 'undefined' is not assignable to type 'string'.